x-pack/elastic-agent/docs/uninstall-elastic-agent.asciidoc

```diff
@@ -19,5 +21,17 @@ stops and uninstalls any managed programs, such as {beats} and
 
 If you run into problems, see <<fleet-troubleshooting>>.
 
+== Uninstall on DEB or RPM
+
+The `uninstall` command is not supported for DEB or RPM installations. To
+uninstall {agent} on DEB or RPM:
+
+. If the agent is managed by {fleet}, <<unenroll-elastic-agent,unenroll the agent>>.
+. On your host, <<stop-elastic-agent,stop the agent>>.
+. Manually remove the agent files from your system.
+
+//REVIEWERS: We need something like this for deb/rpm users. Please confirm the
+//process or suggest alternatives.
```

Comment:
I can't comment on this

Reply:
Since this is beta, I'll leave it as an todo item for the next release.